.github/workflows/remote-copy.yml

```diff
@@ -184,31 +212,42 @@ jobs:
         id: manifest
         env:
           REMOTE_MANIFEST_FILE_LIST_PATH: ${{ vars.REMOTE_TMP_DIR }}/remote-copy-manifests-${{ github.run_id }}.log
+          LOCAL_MANIFEST_FILE_LIST_PATH: ./manifests-copied.log
+          MANIFEST_FILE_NAME: manifest.yaml
         # Generate manifests for files copied over in the earlier rsync job
         run: |
           ssh ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST_DATA }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
           module use ${{ vars.YAMF_MODULE_PATH }}
           module load ${{ vars.YAMF_MODULE_NAME }}
 
           manifest_paths=()
-          while read -r path:
-            cd $(dirname $path)
-            manifest_dir=$(basename $path)
-            yamf add -n manifest.yaml --force $manifest_dir
-            manifest_paths+=("$manifest_dir/manifest.yaml")
-          done <<< "${{ steps.copy.outputs.paths }}"
+          for path in ${{ steps.copy.outputs.paths }}; do
+            echo "Path is $path"
+            manifest_dir=$(dirname $path)
+            cd $manifest_dir || exit
+            manifest_file=$(basename $path)
+            echo "Generating a manifest in $manifest_dir for $manifest_file"
+            yamf add -n ${{ env.MANIFEST_FILE_NAME }} --force $manifest_file
+            manifest_paths+=("$manifest_dir/${{ env.MANIFEST_FILE_NAME }}")
+          done
           echo "${manifest_paths[@]}" > ${{ env.REMOTE_MANIFEST_FILE_LIST_PATH }}
           EOT
 
-          echo "paths=$(scp -q ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST_DATA }}:${{ env.REMOTE_MANIFEST_FILE_LIST_PATH }} /dev/stdout) >> $GITHUB_OUTPUT
+          rsync -e 'ssh -i ${{ steps.ssh.outputs.private-key-path }}' \
+            ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST_DATA }}:${{ env.REMOTE_MANIFEST_FILE_LIST_PATH }} \
+            ${{ env.LOCAL_MANIFEST_FILE_LIST_PATH }}
+          cat ${{ env.LOCAL_MANIFEST_FILE_LIST_PATH }}
+          paths=$(cat ${{ env.LOCAL_MANIFEST_FILE_LIST_PATH }})
+          echo "paths are $paths"
```

Comment:
The paths are being catted to STDOUT twice.

Reply:
Also "paths are $paths" isn't super informative. Is this just debugging?

Reply:
The first bit of feedback was done in https://github.com/ACCESS-NRI/model-config-inputs/pull/3/commits/c9e050456f71712f9e9d5b3ba9c007ce6871cc6f and the second is done in https://github.com/ACCESS-NRI/model-config-inputs/pull/3/commits/7ebfeb03d0843715734f99c4dc4859b615d91eb4